eduvpn/manager.py

```diff
@@ -131,24 +136,26 @@ def delete_provider(uuid):
 
 
 def connect_provider(uuid):
-    """
+   #type: (str) -> None
+	"""
     Enable the network manager configuration by its UUID
-
+    
     args:
         uuid (str): the unique ID of the configuration
     """
-    logger.info("connecting profile with uuid {} using NetworkManager".format(uuid))
-    if not have_dbus():
-        raise EduvpnException("No DBus daemon running")
+	logger.info("connecting profile with uuid {} using NetworkManager".format(uuid))
```

Comment:
this looks like wrong indendation

Reply:
Mypy does not detect it as wrong indentation.
![image](https://user-images.githubusercontent.com/36157555/63028112-76e7cb00-beae-11e9-9f3f-86aa2dc6522b.png)